### ISSUE TITLE
enhance error message when remove a container does not exist

### ISSUE THREAD

**Issue by gouyang** [projectatomic/buildah#411]
`error reading build container: container not known` shows twice in the error message.

```
# buildah rm busybox-working-container-4
error reading build container "busybox-working-container-4": error reading build container: container not known
error reading build container: container not known
```

**Comment by ripcurld0** [comment]
Thanks @gouyang

**Comment by TomSweeneyRedHat** [comment]
@gouyang Just to clarify, the busybox-working-container-4 did not exist when you tried to remove it?  Thanks for the reports, you're keeping us on out toes and I like it!

**Comment by gouyang** [comment]
@TomSweeneyRedHat yes, the busybox-working-container-4 did not exist.

**Comment by ripcurld0** [comment]
My guess is since multiple containers can be given to `buildah-rm` if it failed to remove of them it prints it on the screen and in the end it returns the error. Thus, the duplicate error message.

It shouldn't take much to fix it.

**Comment by rhatdan** [comment]
We handle this very differently in libpod.  Basically code returns lastError, if an Error happens when lastError is set, lastError is printed out , and then replaced with the new lastError.

**Comment by ripcurld0** [comment]
Thanks @rhatdan, let me change that to what libpod does.

**Comment by TomSweeneyRedHat** [comment]
Proposed fix in #458

**Comment by ripcurld0** [comment]
Closing as PR #458 fixes this problem.
Feel free to reopen if it wasn't fully fixed. Thanks.
